### DISCUSSION TITLE
热榜

### DISCUSSION THREAD

**Discussion by nljie1103** [imsyy/DailyHotApi#21] [General]
### 问题描述

[今日热榜 - https://hot.imsyy.top/](https://hot.imsyy.top/)

这个网站是怎么搭建的

**Comment by china-LY** [comment]
在vercel上面直接创建项目然后仓库选作者的github仓库地址就行了

**Comment by nljie1103** [comment]
> 在vercel上面直接创建项目然后仓库选作者的github仓库地址就行了

是的，在vercel上部署后，只有个这页面
![image](https://github.com/imsyy/DailyHotApi/assets/59471244/86e8bec8-818d-449a-8713-ef12b7c08d0b)

**Comment by china-LY** [comment]
那是因为你没去看https://github.com/imsyy/DailyHot     这个项目
